include/zephyr/sys/__assert.h

```diff
@@ -137,15 +137,15 @@ void assert_post_action(const char *file, unsigned int line);
 #warning "__ASSERT() statements are ENABLED"
 #endif
 #else
-#define __ASSERT(test, fmt, ...) { }
+#define __ASSERT(test, fmt, ...) {(void)test}
```

Comment:
This does change the behavior too, but I guess this is fine?

We should add parentheses though.

```suggestion
#define __ASSERT(test, fmt, ...) {(void)(test)}
```